clang/lib/CAS/CASOptions.cpp

```diff
@@ -116,7 +109,7 @@ llvm::Error CASOptions::initCache() const {
 
   SmallString<256> PathBuf;
   if (CASPath == "auto") {
-    getClangDefaultCachePath(PathBuf);
+    getDefaultOnDiskCASPath(PathBuf);
```

Comment:
I suggest you separately cherry-pick 5cef7a629 instead of merging it into this single commit, since it's not really MCCAS-specific.